activities/Fototoon.activity/js/toon.js

```diff
@@ -1289,6 +1289,8 @@ define(["easel","sugar-web/datastore","sugar-web/env","webL10n","humane"], funct
         };
 
         this.createControls = function() {
+	    this._stage.addChild(this._resizeButton);
```

Comment:
Warning: indentation is wrong on this line, it mix TAB and SPACE.

Reply:
I've made the indentation proper now

Reply:
okay so move these lines after the drawing of dotted lines, it should work for title too

Reply:
It didn't change on applying it to the back there

Reply:
Yes but if you want to fix the issue regarding main title, these lines should be moved.